sdk/scheduler/src/test/java/org/apache/mesos/offer/NamedVIPPortRequirementTest.java

```diff
@@ -0,0 +1,22 @@
+package org.apache.mesos.offer;
+
+import org.apache.mesos.Protos;
+import org.apache.mesos.testutils.TestConstants;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class NamedVIPPortRequirementTest {
+
+    @Test
+    public void testNamedVIPPortRequirementConstruction() throws NamedVIPPortRequirement.NamedVIPPortException {
```

Comment:
At least one test that actually throws the `NamedVIPPortException` is probably wanted.